multiqc/modules/megahit/megahit.py

```diff
@@ -0,0 +1,146 @@
+                        data[f["s_name"]]["megahit_avg_contig"] = int(match.group(5))
+                        data[f["s_name"]]["megahit_n50"] = int(match.group(6))
+
+        # Filter to strip out ignored sample names
+        data = self.ignore_samples(data)
+        if len(data) == 0:
+            raise ModuleNoSamplesFound
+        log.info(f"Found {len(data)} reports")
+
+        # Write parsed report data to a file
+        self.write_data_file(data, f"multiqc_{self.anchor}")
+
+        headers = {
+            "megahit_mem": {
+                "title": "Memory",
+                "description": "Memory used",
+                "min": 0,
+                "scale": "Reds",
+                "format": lambda x: humanize.naturalsize(x),
+            },
+            "megahit_time": {
+                "title": "Time",
+                "description": "Time elapsed",
+                "min": 0,
+                "suffix": "&nbsp;s",
+                "scale": "Greys",
+            },
+            "megahit_contigs": {
+                "title": "Contigs",
+                "description": "Number of contigs",
+                "min": 0,
+                "scale": "Greens",
+                "hidden": True,
+                "format": "{:,d}",
+            },
+            "megahit_bases": {
+                "title": f"Bases, {config.base_count_prefix}",
+                "description": f"Total number of bases ({config.base_count_desc})",
+                "shared_key": "base_count",
+                "min": 0,
+                "scale": "Blues",
+                "hidden": True,
+            },
+            "megahit_min_contig": {
+                "title": "Min contig",
+                "description": "Minimum contig length",
+                "min": 0,
+                "scale": "YlGn",
+                "hidden": True,
+                "suffix": "&nbsp;bp",
+                "format": "{:,d}",
+            },
+            "megahit_max_contig": {
+                "title": "Max contig",
+                "description": "Maximum contig length",
+                "min": 0,
+                "scale": "YlGn",
+                "hidden": True,
+                "suffix": "&nbsp;bp",
+                "format": "{:,d}",
+            },
+            "megahit_avg_contig": {
+                "title": "Avg",
+                "description": "Average contig length",
+                "min": 0,
+                "scale": "YlGn",
+                "hidden": True,
+                "suffix": "&nbsp;bp",
+                "format": "{:,d}",
+            },
+            "megahit_n50": {
+                "title": "N50",
+                "description": "N50 contig length",
+                "min": 0,
+                "scale": "Greens",
+                "suffix": "&nbsp;bp",
+                "format": "{:,d}",
+            },
+        }
+
+        # Basic Stats Table
+        self.general_stats_addcols(data, headers)
```

Comment:
I'm not sure about just sticking everything in general stats, even if much of it is hidden. I'd prefer to just put in the columns that we think people will want to compare.

I'd also personally think that memory and time are two of the least interesting columns, despite what the OP said. Is that just me? 🙄  I would have probably gone for bases / contigs / N50?

Reply:
Yeah, I agree with you :)